src/prefect/events/schemas/automations.py

```diff
@@ -429,4 +432,112 @@ class AutomationCore(PrefectBaseModel, extra="ignore"):
 
 
 class Automation(AutomationCore):
-    id: UUID = Field(..., description="The ID of this automation")
+    id: Optional[UUID] = Field(default=None, description="The ID of this automation")
+
+    @classmethod
+    @sync_compatible
+    async def create(cls: Type[Self], automation: Type[Self]) -> UUID:
+        """
+        Create a new automation.
+
+        auto_to_create = Automation(
+        name="woodchonk",
+        trigger=EventTrigger(
+            expect={"animal.walked"},
+            match={
+                "genus": "Marmota",
+                "species": "monax",
+            },
+            posture="Reactive",
+            threshold=3,
+            within=timedelta(seconds=10),
+        ),
+        actions=[CancelFlowRun()]
+        )
+        created_automation = Automation.create(auto_to_create)
+        """
+        client, _ = get_or_create_client()
+        automation = AutomationCore(**automation.dict(exclude={"id"}))
+        automation = await client.create_automation(automation=automation)
+        return automation if automation else None
+
+    @classmethod
+    @sync_compatible
+    async def update(
+        cls: Type[Self], automation_id: UUID, automation: Type[Self]
```

Comment:
i think all the `automation_id` kwargs should just be `id`

Reply:
yep these are gone now anyway